signac/contrib/import_export.py

```diff
@@ -293,25 +84,18 @@ def _export_jobs(jobs, path, copytree):
     dst : str
         Destination path.
 
-    Raises
-    ------
-    RuntimeError
-        If paths generated with given path function are not unique.
-
     """
     # Transform the path argument into a callable if necessary.
     if callable(path):
         path_function = path
+        _check_path_function(jobs, path_spec=path, path_function=path_function)
     else:
         path_function = _make_path_function(jobs, path)
+        # path_function is checked inside _make_path_function
```

Comment:
Seems like `_make_path_function` can be used in place of the logic here. We should be able to avoid having two paths (`if callable(path)` in the snippet above) that independently call `_check_path_function`, since `_make_path_function` can take a callable. Might need to refactor `_make_path_function` to validate at the end. See other comment.
```suggestion
    path_function = _make_path_function(jobs, path)
```